modules/cnv-cloning-local-volume-to-another-node.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-moving-local-vm-disk-to-different-node.adoc
+
+[id="cnv-cloning-local-volume-to-another-node_{context}"]
+= Cloning a local volume to another node
+
+You can move a virtual machine so that it runs on a specific node by cloning the underlying PersistentVolumeClaim (PVC).
```

Comment:
You can move a virtual machine __disk__ so that it resides on a specific node by cloning the underlying PersistentVolumeClaim (PVC)